spec/tictactoe_spec.rb

```diff
@@ -1,26 +1,123 @@
 require 'spec_helper' 
 require 'tictactoe' 
+## Faker Classes 
+class FakeIO
+  attr_accessor :stdin, :stdout 
+  def initialize(stdin: StringIO.new('') ) 
+      @stdin = stdin 
+      @stdout = []  
+  end 
+
+  def puts( string ) 
+    @stdout << string     
+  end 
+
+  def gets 
+    @stdin.gets @stdin.string 
+  end 
+end 
+
+def isRowOutputed( io: , row:  ) 
+    expect( io.stdout.include? row).to eql true 
+end 
```

Comment:
Hey this is cool.

As I mentioned before, the convention would be to use snake case, so `is_row_outputed`.